### ISSUE TITLE
Feature request: 16x9 paper sizes

### ISSUE THREAD

**Issue by WPettersson** [xournalpp/xournalpp#249]
tl;dr A 16:9 page size option as a selectable option, instead of typing measurements.

I rarely use xournal to actually create documents from scratch that I intend to print out physically. I'm either annotating PDFs (which have a given page size already) or I'm just doodling on my laptop (or even creating slides for presentations). In these last 2 cases, I often just want to "fill" the page. Zooming does somewhat work, but if I ever export to PDF to send notes to someone, the pages no longer line up with what I was doing. I currently get around this by setting a 16:9 page size (actually 32cm x 18cm) but this is probably something that other people do as well.

Other options include adding 4:3 or 16:10 options as well.

I might get around to coding this myself, I've had a bit of a look and it should be "relatively" straight forward.

**Comment by andreasb242** [comment]
Currently we use GTK Formats, and they do not support 19:9.

Default format should be added at:
Settings::loadDefault()

Like:
	this->visiblePageFormats = GTK_PAPER_NAME_A4 "," GTK_PAPER_NAME_A5 ","
							   GTK_PAPER_NAME_LETTER "," GTK_PAPER_NAME_LEGAL ","
							   "custom_16:9_1600x900";
or something like this (1600x900 should be dot's or so, only as example)

In FormatDialog::FormatDialog the values are loaded.
After the loop loading the standard formats, 

	for (GList* l = list; l != NULL; l = next)
	{
		GtkPaperSize* s = (GtkPaperSize*) l->data;
...	}

HERE

Split formatlist by comma, and load all formats starting with "custom".

**Comment by andreasb242** [comment]
I made the formats hardcoded, but it can easy be made configurable, but at the moment I think it's not needed.
